docs/docusaurus/README.md

```diff
@@ -11,15 +11,15 @@ https://docusaurus.io/docs/installation#requirements
 
 ## Installation
 
-From the repo root run:
+From this folder (docs/docusaurus) run:
 
 ```console
 yarn install
 ```
 
 ## Local Development
 
-For the fastest iterative dev loop, start a local server and open up the compiled site in a browser window. Most changes are reflected live without needing server restarts.
+For the fastest iterative dev loop, start a local server and open up the compiled site in a browser window. Most changes are reflected live without needing server restarts. Run this and all commands in this folder.
```

Comment:
Which folder?  Since we're in a new section, we need to specify it again.

```suggestion
For the fastest iterative dev loop, start a local server and open up the compiled site in a browser window. Most changes are reflected live without needing server restarts. Run this and all commands in the `/docs/docusaurus/` folder.
```